File: question3.c
#include <stdio.h>
//
// Created by hp on 26-05-2025.
//
//int main() {
    void todo1(){
        FILE *f;
        f = fopen("todo1.txt","w");
        int x;
        printf("enter number of tasks: ");
        scanf("%d",&x);
        for (int i = 0; i < x; i++) {
            char task[100];
            printf("enter task: ");
            scanf("%s",task);
            fprintf(f,"%s\n", task);
        }
        fclose(f);
    }
    void todo2(){
        FILE *f;
        f = fopen("todo2.txt","w");
        int y;
        printf("enter number of users: ");
        scanf("%d",&y);
        for (int i = 0; i < y; i++) {
            char user[100];
            printf("enter user: ");
            scanf("%s",user);
            fprintf(f,"%d. %s \n",i+1, user);
            int x;
            printf("enter number of tasks: ");
            scanf("%d",&x);
            for (int i = 0; i < x; i++) {
                char task[100];
                printf("enter task: ");
                scanf("%s",task);
                fprintf(f,"%s\n", task);
            }
        }
        fclose(f);
    }
//}

File: question 3.py
#to do list
def todo():
    f=open('todo.txt','w')
    x=int(input("enter the number of tasks to be done: "))
    for i in range(x):
        f.write(str(i)+". "+input("enter the task: ")+"\n")

def todo2():
    f=open('todo2.txt','w+')
    x=int(input("enter the number of users: "))
    for i in range(x):
        name=input("enter the user name: ")
        g=[name+":"+'\n']
        print("enter tasks and put in . to stop")
        u=1
        while True:
            k=input("Enter the task: ")
            if k !=".":
                g.append(str(u)+". "+str(k)+'\n')
            else:
                break
            u+=1
        f.writelines(g)
        f.write("\n")